app/services/push_notification/account_suspended_event.rb

```diff
@@ -0,0 +1,30 @@
+module PushNotification
+  class AccountSuspendedEvent
+    EVENT_TYPE = 'https://schemas.openid.net/secevent/risc/event-type/account-disabled'.freeze
+
+    attr_reader :user
+
+    def initialize(user:)
+      @user = user
+    end
+
+    def event_type
+      EVENT_TYPE
+    end
+
+    def payload(iss_sub:)
+      {
+        subject: {
+          subject_type: 'iss-sub',
+          iss: Rails.application.routes.url_helpers.root_url,
+          sub: iss_sub,
+        },
+        reason: 'account-suspension',
```

Comment:
not sure if we can add any other reason here.
![image](https://github.com/18F/identity-idp/assets/109746710/5ab38ab6-2dbf-476b-a331-27110cc105a1)

Reply:
Since that is not one of the known values, and it's optional, I would leave it out for now